flybywiresim/a32nx
# Asobo 1.10.11

Fixes #[issue_no]

## Summary of Changes


## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
Add a selection in the control menu for Engage managed altitude mode
